src/openrct2/Game.h

```diff
@@ -95,7 +95,6 @@ enum class GameCommand : int32_t
     ModifyTile,
     EditScenarioOptions,
     PlacePeepSpawn,
-    SetClimate,
```

Comment:
This will definitly need a network version bump this would change all actions after this!

Reply:
Thanks. But not replay recordings, it seems! Amusingly, that's what I was most worried about.

Reply:
I suppose none of them are tested in the replays. 🙈

Reply:
That makes a lot of sense, actually 😅 Time for more tests!